### ISSUE TITLE
myyna configuration

### ISSUE THREAD

**Issue by kacemlight** [cubettech/myyna#11]
Hello,

I installed myyna and start it succefully, but y´when I try to configure it I get an error : Sorry!

On the terminal I get : 

[TypeError: Cannot call method 'authenticate' of null]
Sorry! Installation Failed!
Reverting changes..

Can you please help to resolve this problem ?

**Comment by jmtt89** [comment]
yes this error is because your update the mongoDB library and the implementation has change

the dependencies of this projects are sooo outdated

I made a pull request to solve all this outdated bugs all backend are working fine with last version of each library now ^_^
